__tests__/server.test.js

```diff
@@ -71,4 +72,41 @@ describe("GET /api/articles/:article_id", ()=>{
             expect(body.msg).toBe("Bad request")
         })
     })
+})
+
+describe("GET: /api/articles/:article_id/comments", ()=>{
+    test("200 - Getting all comments for a related article",()=>{
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body: {comments}}) => {
+            expect(comments).toHaveLength(11)

```

Comment:
Could you add a check to make sure these are ordered by the mosr recent comment first?